modules/primer-marketing-utilities/docs/borders.md

```diff
@@ -17,3 +17,22 @@ Top, right, bottom, and left border utilities are can be used responsively to ad
   .border-top-0
 </div>
 ```
+
+## Border Colors
+
+### White border with alpha transparency
+
+In addition to `.border-black-fade` which is in primer-core, primer-marketing includes `.border-white-fade`. This adds a rgba white border with an alpha transparency of `0.15`. This is useful when you want a border that is a lighter shade of the background color.
```

Comment:
I think starting off with what this style is and how it can be used is more helpful, and I don't think you need to specifically reference the `border-black-fade` specifically but would be nice to link to the border color utilities. If you do think it's important to mention the black version, perhaps say "such as `border-black-fade` in the last sentence? Feel free to edit:

> Use `border-white-fade` to add a white border with an alpha transparency of `0.15`. This is useful when you want a border that is a lighter shade of the background color. Additional border colors are available in [primer-core border utilities](/borders/#border-colors).

(I think this is the right url but worth double checking in the style guide.)